example_az_deploy.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+import os
+from configparser import ConfigParser
+
+# This script will deploy the cdf-fabric-replicator container to Azure Container Instances. 
+
+# To configure the script, create a .env file with the following and define the values with the appropriate values for your environment:
+
+# COGNITE_PROJECT=
+# COGNITE_BASE_URL=
+# COGNITE_TOKEN_URL=
+# COGNITE_CLIENT_SECRET=
+# COGNITE_CLIENT_ID=
+# COGNITE_STATE_DB=
+# COGNITE_STATE_TABLE=
+# COGNITE_EXTRACTION_PIPELINE=
+# LAKEHOUSE_BASE_URL=
```

Comment:
Not sure if line 17 and 18 are still needed. Also if there are other parameters are added to the env variables, they should also be added here.

Reply:
we will have to sort that out when we want to test deploy in az